schemas/ajv.absolutePath.js

```diff
@@ -16,13 +18,28 @@ module.exports = (ajv) => ajv.addKeyword("absolutePath", {
 	errors: true,
 	type: "string",
 	compile(expected, schema) {
-		function callback(data) {
-			const passes = expected === /^(?:[A-Za-z]:\\|\/)/.test(data);
-			if(!passes) {
+		const callback = expected ? data => {
+			if (path.isAbsolute(data)) {
```

Comment:
Please don't use `path.isAbsolute` since it's behavior is platform-dependent. Instead use a regexp like in the original code.